scripts/install.tpl.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+olm_version=$OLM_V0_VERSION
```

Comment:
These are "supposed" to be set from the envsubst command in the makefile's `quickstart` target but I added some simple checks just in case.